styles/plugins/linter.less

```diff
@@ -5,6 +5,11 @@
   }
 }
 
+#linter-inline a.eslint {
+  background: #fff;
```

Comment:
change this to @seti-primary-text to make the background color dynamic and then I'm cool to merge this in.